js/controller/BoardController.js

```diff
@@ -200,6 +200,27 @@ app.controller('BoardController', function ($rootScope, $scope, $stateParams, St
 		CardService.archive(card);
 		StackService.removeCard(card);
 	};
+    $scope.isCurrentUserAssigned = function (card) {
```

Comment:
Please use tabs instead of spaces.

Reply:
fixed by 02c1807f622e81212a97843779a90ed9641e9a7c